cmake/FindSpEC.cmake

```diff
@@ -45,6 +45,29 @@ find_package(MPI COMPONENTS C)
 
 if (SPEC_PACKAGED_EXPORTER_LIB AND SPEC_EXPORTER_FACTORY_OBJECTS AND
     SPEC_EXPORTER_INCLUDE_DIR AND MPI_C_FOUND)
+
+  # Deal with FFTW3
+  #
+  # If it was dynamically linked into SpEC then we need to dynamically link it
+  # into SpECTRE.
+  set(_MACHINE_DEF_FILE
+    ${SPEC_ROOT}/MakefileRules/this_machine.def)
+  set(_FFTW3_LINK_FLAGS "")
+  if (EXISTS ${_MACHINE_DEF_FILE})
+    file(STRINGS ${_MACHINE_DEF_FILE} _MACHINE_DEF)
+    string(REGEX MATCH "FFTW_LIB[^;]+"
```

Comment:
On Wheeler this matches a commented-out line in the machine file:

```sh
#-----------------------
# FFTW3
#-----------------------
FFTW_INC = -I$(FFTW_HOME)/include
#FFTW_LIB = -Wl,-rpath,$(FFTW_HOME)/lib -L$(FFTW_HOME)/lib -lfftw3
FFTW_LIB = /usr/lib64/libfftw3.a
```

I haven't found a quick way to match at the beginning of lines in multiline strings.

Reply:
Sigh... Easy fix, just filter out commented lines :) Thanks for catching this!